pkg/stacks/unpack.go

```diff
@@ -44,14 +44,19 @@ const (
 	resourceFileNamePattern = "*resource.yaml"
 	groupFileName           = "group.yaml"
 	appFileName             = "app.yaml"
-	iconFileNamePattern     = "icon.*"
-	crdFileNamePattern      = "*crd.yaml"
-	uiSpecFileNamePattern   = "*ui-schema.yaml"
-	uiSpecFileName          = "ui-schema.yaml" // global ui schema filename
-	yamlSeparator           = "\n---\n"
+
+	// iconFileNamePattern is the pattern used when walking the stack package and looking for icon files.
+	// Icon files that are for a single resource can be prefixed with the kind of the resource, e.g.,
+	// mytype.icon.svg
+	// Multiple icon types and therefore file extensions are supported: svg, png, jpg, gif
+	iconFileNamePattern = "*icon.*"
```

Comment:
This seems potentially a little **too** fuzzy in the long run . . . I'm not thinking of a *good* example, but what if I have a `decepticon.crd.yaml`?

Reply:
Good point, thank you for bringing up that edge case.  Sounds like in the long term we'd want to use more powerful regular expressions instead of the more simple `filepath.Match` we are using right now.